src/EditorFeatures/Core/Implementation/Outlining/OutliningSpan.cs

```diff
@@ -31,13 +32,22 @@ internal class OutliningSpan
         /// </summary>
         public bool IsDefaultCollapsed { get; }
 
-        public OutliningSpan(TextSpan textSpan, TextSpan hintSpan, string bannerText, bool autoCollapse, bool isDefaultCollapsed = false)
+        public string Type { get; }
+
+        public OutliningSpan(TextSpan textSpan, TextSpan hintSpan, string bannerText, bool autoCollapse, bool isDefaultCollapsed)
```

Comment:
It's not.  That API is left around for *binary* compat.  The new constructor will then be called if you recompile